ocaml/tests/test_observer.ml

```diff
@@ -187,10 +483,19 @@ let test_attribute_validation () =
 
 let test =
   [
-    ("test_hashtbl_leaks", `Quick, test_hashtbl_leaks)
-  ; ("test_tracing_exn_backtraces", `Quick, test_tracing_exn_backtraces)
+    ( "test_observer_create_and_destroy"
+    , `Quick
+    , test_observer_create_and_destroy
+    )
+  ; ("test_observer_valid_params", `Quick, test_observer_valid_params)
+  ; ("test_observer_endpoint", `Quick, test_observer_endpoint)
+  ; ("test_file_export", `Quick, test_file_export_writes)
```

Comment:
This will fail first time as the directory would not have been created as the export loop would not have taken place. This test also ends up leaving a next of empty directories in my repo. I don't know too much about xenRT testing this test feels better here as it is quite stateful.